batch/extension-jberet/src/main/java/org/wildfly/extension/batch/jberet/BatchSubsystemDefinition.java

```diff
@@ -84,7 +84,7 @@
     static final SimpleAttributeDefinition DEFAULT_THREAD_POOL = SimpleAttributeDefinitionBuilder.create("default-thread-pool", ModelType.STRING, true)
             .setAllowExpression(false)
             .setAttributeGroup("environment")
-            .setAttributeMarshaller(NameAttributeMarshaller.INSTANCE)
+            .setAttributeMarshaller(NameAttributeMarshaller.INSTANCE) //todo this looks like it is missing capability reference
```

Comment:
But it does reference a service itself.
Proper solution would probably be to add support for capabilities to Threads subsystem.